client/src/services/ApiService.js

```diff
@@ -85,7 +85,7 @@ class ApiService {
     await this.initialize();
 
     let count = 1;
-    for await (const page of this.client.queryTwins(query).byPage()) {
+    for await (const page of this.client.queryTwins(query.replace("'", "")).byPage()) {//eslint-disable-line
```

Comment:
I'm not 100% sure if removing all quotes is the right option here. Couldn't there be valid uses of a single quote? Should we be escaping this somehow instead?

Reply:
This one shouldn't be here I make a lot of this changes in order to check where the fail was presented. I already rolled this one back